src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -334,7 +334,7 @@ public function addNode(\DOMNode $node)
      *
      * @param int $position The position
      *
-     * @return self
+     * @return static
```

Comment:
💡 Most methods in this class return the result of `createSubCrawler()` which is already (correctly) annotated with `@return static`.